api_docs/include/rest-endpoints.md

```diff
@@ -106,6 +106,11 @@
 * [Create a custom profile field](/api/create-custom-profile-field)
 * [Update realm-level defaults of user settings](/api/update-realm-user-settings-defaults)
 
+#### Invitations
```

Comment:
Looking at the sidebar sections, maybe we should put this one after the **Users** section, but before the **Server and organizations** section.